widget-store/public/app.html

```diff
@@ -0,0 +1,577 @@
+                  <span x-text="product.product"></span> &#8212; Only
+                  <b x-text="product.inventory"></b> left!
+                </div>
+                <p x-text="product.description"></p>
+                <button
+                  class="bg-green-500 border-none text-white py-4 px-6 text-center text-2xl cursor-pointer rounded-lg transition duration-300 mt-5 hover:bg-green-600 hover:shadow-lg"
+                  id="purchaseButton"
+                  x-on:click="purchase(uuid)"
+                >
+                  Buy Now for $<span x-text="product.price"></span>
+                </button>
+              </div>
+            </template>
+
+            <template x-if="showPaymentView">
+              <div
+                class="text-center flex flex-col items-center justify-center h-full"
+              >
+                <div class="text-xl font-semibold mb-4">
+                  Do you want to confirm payment?
+                </div>
+                <div class="space-x-4">
+                  <button
+                    x-bind:disabled="connectionLost"
+                    class="bg-green-600 hover:bg-green-700 text-white font-bold py-2 px-4 rounded disabled:bg-gray-500"
+                    x-on:click="sendPaymentRequest('paid')"
+                  >
+                    Confirm Payment
+                  </button>
+                  <button
+                    x-bind:disabled="connectionLost"
+                    class="bg-red-600 hover:bg-red-700 text-white font-bold py-2 px-4 rounded disabled:bg-gray-500"
+                    x-on:click="sendPaymentRequest('failed')"
+                  >
+                    Deny Payment
+                  </button>
+                </div>
+              </div>
+            </template>
+
+            <template x-if="showOrderStatusView">
+              <div x-data="retrieveOrder(orderId)">
+                <div class="p-4">
+                  <div class="text-lg font-semibold mb-2">
+                    Order ID: <span x-text="order.id"></span><br />Status:
+                    <span x-text="order.status"></span><br />
+                    <span x-text="order.last_update_time"></span>
+                  </div>
+                  <button
+                    x-on:click="resetView()"
+                    class="mt-4 bg-green-500 text-white py-2 px-4 rounded-lg hover:bg-green-600"
+                  >
+                    Continue Shopping
+                  </button>
+                </div>
+              </div>
+            </template>
+          </div>
+        </div>
+
+        <div :class="visiblePanelOnMobile === 'tools' || 'hidden md:block'">
+          <div class="flex flex-col gap-6 p-5 text-center">
+            <h2 class="text-3xl text-center">Server Tools</h2>
+            <div x-html="divider"></div>
+            <p class="text-left text-gray-500">
+              Crash the application at any time to simulate a server failure.
```

Comment:
I'd condense both paragraphs to something much shorter like this:

Crash the application at any time to simulate a server failure or service interruption. After crashing, the application will momentarily become inaccessible but then resume from exactly where it left off before the crash. Try it and see!